ank/src/cli_commands/server_connection.rs

```diff
@@ -402,15 +405,20 @@ mod tests {
         );
         sim.will_send_response(
             REQUEST,
-            ResponseContent::CompleteState(Box::new(complete_state(WORKLOAD_NAME_1))),
+            ank_base::response::ResponseContent::CompleteState(
+                complete_state(WORKLOAD_NAME_1).into(),
```

Comment:
Maybe we should change the complete_state() fn to return an ank_base complete state directly.

Reply:
The instances where common::objects is created and needed greatly outnumber the ank_base ones. Instead I've proceeded to use proto_complete_state.